taichi/ir/statements.h

```diff
@@ -858,13 +858,21 @@ class IfStmt : public Stmt {
 class PrintStmt : public Stmt {
  public:
   using EntryType = std::variant<Stmt *, std::string>;
-  std::vector<EntryType> contents;
+  using FormatType = std::optional<std::string>;
+  const std::vector<EntryType> contents;
+  const std::vector<FormatType> formats;
 
   explicit PrintStmt(const std::vector<EntryType> &contents_)
```

Comment:
Any place still using this legacy constructor after your patch? Otherwise we should consider removing it.

Reply:
Old constructor was removed~